src/core/load-fragment.c

```diff
@@ -2745,7 +2747,28 @@ int config_parse_syscall_filter(
                         break;
                 }
 
-                r = syscall_filter_parse_one(unit, filename, line, c, invert, word, true);
+                /* Optionally, errno is specified. */
+                if (p && p[-1] == ':') {
+                        r = extract_first_word(&p, &e, NULL, 0);
+                        if (r == -ENOMEM)
+                                return log_oom();
+                        if (r < 0) {
+                                log_syntax(unit, LOG_WARNING, filename, line, r, "Invalid syntax, ignoring: %s", rvalue);
+                                break;
+                        }
+                        if (r == 0) {
+                                log_syntax(unit, LOG_WARNING, filename, line, r, "Missing argument after ':', ignoring: %s", rvalue);
+                                break;
+                        }
+
+                        num = parse_errno(e);
+                        if (num < 0) {
+                                log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to parse error number, ignoring: %s", rvalue);
+                                return 0;
+                        }
+                }
+
```

Comment:
Is ignoring errors here the right approach ?  we ignore errors in case we can't resolve syscall names and groups, maybe here we should not ignore the syscall name and just go then with default value of SystemCallError ?

Reply:
OK. I will update.